src/Template/Bake/Template/index.ctp

```diff
@@ -16,6 +16,7 @@
 <?php
 /**
   * @var \<%= $namespace %>\View\AppView $this
+  * @var \<%= $namespace %>\Model\Entity\<%= $entityClass %>[] $<%= $pluralVar %>
```

Comment:
@ndm2 That is a useless annotation, though. You need the specific entity, otherwise all the properties are yellow (not findable), and so all relations with it.

See my ide helper plugin, and this here works right into that.